revamped bitstory UI

sameer2800 opened the pull request:
1)used bootstrap libraries to build form
2) also tested on mobile phone. works flawlessly,adjusts automatically on mobile phone. 
3)attaching the UI screenshots.
<img width="1440" alt="screen shot 2018-09-29 at 4 05 08 pm" src="https://user-images.githubusercontent.com/8151356/46244893-6ef07b00-c403-11e8-8c9d-564a8868ff5e.png">
<img width="764" alt="screen shot 2018-09-29 at 4 06 21 pm" src="https://user-images.githubusercontent.com/8151356/46244899-74e65c00-c403-11e8-9802-2d5817bb4c38.png">
<img width="637" alt="screen shot 2018-09-29 at 4 06 54 pm" src="https://user-images.githubusercontent.com/8151356/46244901-7f085a80-c403-11e8-9432-38ee49533ea7.png">

Commit:
css changes

Commit:
used bootstrap for html changes

sameer2800 commented:
this PR fixes #1

JoshOrndorff commented:
This looks amazing. Thank you very much! I want to request a couple little changes before I ultimately accept this beautiful new interface.

1. I see you included jquery, but did you ever actually use it? I removed that line locally and it seems to be unnecessary
2. Rather than hitting a CDN, can the bootstrap be included. Either just downloaded and committed, or preferably by using npm?

Excited to wrap this one up.

Commit:
bootstrap library files

Commit:
removed jquery, loading bootstrap from local files

Commit:
removed unwanted files

Commit:
removed unncessary bootstrap files

sameer2800 commented:
thanks for much appreciation. As you suggested, i made the changes mentioned.
1)removed jquery plugin.earlier i added by mistake.
2)added bootstrap library files  to the repo itself. 
also i considered adding bootstrap through npm.but after reading this, i don't see any purpose. Even if we add, it will be in a similar way like I did now.
 https://stackoverflow.com/questions/26773767/purpose-of-installing-twitter-bootstrap-through-npm
